sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallAutomationAsyncClient.java

```diff
@@ -287,40 +315,48 @@ Mono<Response<Void>> redirectCallWithResponseInternal(String incomingCallContext
      * Reject a call
      *
      * @param incomingCallContext The incoming call context.
-     * @param callRejectReason The reason why call is rejected. Optional
      * @throws CallingServerErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return Response for a successful CreateCallConnection request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Void> rejectCall(String incomingCallContext, CallRejectReason callRejectReason) {
-        return rejectCallWithResponse(incomingCallContext, callRejectReason).flatMap(FluxUtil::toMono);
+    public Mono<Void> rejectCall(String incomingCallContext) {
```

Comment:
CallRejectReason is not optional in .NET, we should discuss this and keep it consistent in both languages.

Reply:
However, from the model that generated by autorest, it is not a required field of the RejectCallRequest. Do we need this to be required in the service side? I remember in the API review there was a comment about this to be optional.
https://github.com/Azure/azure-sdk-for-java/blob/5eecf3fde3202c4d8722f5d6ba01ee84333088f1/sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/RejectCallRequestInternal.java#L22